src/radiation/radiation_system.hpp

```diff
@@ -1693,6 +1713,11 @@ void RadSystem<problem_t>::AddSourceTerms(array_t &consVar, arrayconst_t &radEne
 					amrex::Gpu::Atomic::Add(p_num_failed_coupling_local, 1);
 				}
 
+				// update iteration counter: (+1, +ite, max(self, ite))
+				amrex::Gpu::Atomic::Add(&p_iteration_counter_local[0], 1);     // total number of radiation updates
```

Comment:
Do we need a separate counter for this? This should be equal to the number of cells, no?